testfiles/PythonPluginCustomSchedule.py

```diff
@@ -6,6 +6,12 @@ class HeatingSetPoint(EnergyPlusPlugin):
     def actuate(self, state, x):
         self.api.exchange.set_actuator_value(state, self.data['actuator_heating'], x)
 
+    def on_begin_zone_timestep_before_set_current_weather(self, state) -> int:
+        if self.api.exchange.warmup_flag(state):
+            return 0
+        self.api.exchange.tomorrow_weather_outdoor_dry_bulb_at_time(state, 3, 2)
+        return 0
```

Comment:
This demonstrates the new calling point being called in a Python Plugin example file, and also demonstrates calling into one of the new weather worker functions.